triton/scripts/build_librispeech_pruned_transducer_stateless7_offline.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+stage=0
+stop_stage=2
+
+# change to your own model directory
+pretrained_model_dir=/mnt/samsung-t7/wend/github/icefall/egs/librispeech/ASR/pruned_transducer_stateless7/exp/
+model_repo_path=./zipformer/model_repo_offline
+
+# modify model specific parameters according to $pretrained_model_dir/exp/onnx_export.log
+VOCAB_SIZE=500
+
+DECODER_CONTEXT_SIZE=2
+DECODER_DIM=512
+
+ENCODER_LAYERS=12
+ENCODER_DIM=512
+CNN_MODULE_KERNEL=31
+
+if [ -d "$pretrained_model_dir/data/lang_char" ]
+then
+    echo "pretrained model using char"
+    TOKENIZER_FILE=$pretrained_model_dir/data/lang_char
+else
+    echo "pretrained model using bpe"
+    TOKENIZER_FILE=$pretrained_model_dir/data/lang_bpe_500/bpe.model
+fi
+
+MAX_BATCH=512
+# model instance num
+FEATURE_EXTRACTOR_INSTANCE_NUM=2
+ENCODER_INSTANCE_NUM=2
+JOINER_INSTANCE_NUM=1
+DECODER_INSTANCE_NUM=1
+SCORER_INSTANCE_NUM=2
+
+
+icefall_dir=/mnt/samsung-t7/wend/asr/icefall
+export PYTHONPATH=$PYTHONPATH:$icefall_dir
+recipe_dir=$icefall_dir/egs/librispeech/ASR/pruned_transducer_stateless7
+
+if [ ${stage} -le -2 ] && [ ${stop_stage} -ge -2 ]; then
+  if [ -d "$pretrained_model_dir" ]
+  then
+    echo "skip download pretrained model"
+  else
+    pushd $icefall_dir/egs/librispeech/ASR/
+    GIT_LFS_SKIP_SMUDGE=1 git clone https://huggingface.co/csukuangfj/icefall-asr-librispeech-pruned-transducer-stateless7-2022-11-11
+    pushd icefall-asr-librispeech-pruned-transducer-stateless7-2022-11-11
+    git lfs pull --include "exp/epoch-30.pt"
+    git lfs pull --include "data/lang_bpe_500/bpe.model"
+    popd
+  fi
+fi
+
+if [ ${stage} -le -1 ] && [ ${stop_stage} -ge -1 ]; then
+    echo "export onnx"
+    cd ${recipe_dir}
+    ./export_onnx.py \
+        --exp-dir ${pretrained_model_dir}/exp \
+        --tokenizer-file $TOKENIZER_FILE \
+        --epoch 30 \
+        --avg 1 \
+        --use-averaged-model 0
+    cd -
+fi
+
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
+     echo "auto gen config.pbtxt"
+     dirs="encoder decoder feature_extractor joiner scorer transducer"
+
+     if [ ! -d $model_repo_path ]; then
+        echo "Please cd to $model_repo_path"
+        exit 1
+     fi
+
+     cp -r $TOKENIZER_FILE $model_repo_path/scorer/
```

Comment:
```suggestion
     cp $TOKENIZER_FILE $model_repo_path/scorer/
```

No need to use `-r` to copy a single file.

Reply:
It's safe to use -r here. Since for Chinese model with LG decoding method, we may need set this TOKENIZER_FILE=$pretrained_model_dir/data/lang_char. Perhaps change the name tokenizer_file to tokenizer_dir_or_file is less confusing.

Reply:
Yes, please rename the variable.